user_service/internal/repository/user_repository.go

```diff
@@ -61,6 +61,16 @@ func (repo *UserRepository) GetUserByEmail(userEmail string) (*models.User, erro
 	return user, nil
 }
 
+func (repo *UserRepository) ValidateUserExistance(userEmail string) (*models.User, error) {
+	user := &models.User{}
+	err := repo.dbClient.Debug().Where("email = ?", userEmail).First(user).Error
+	if err != nil {
+		slog.Info("user doesn't exist's: %v\n", err.Error())
```

Comment:
Totusi logam eroarea nu numai in controlere ? Pur si simplu daca o logam peste tot se va repeta dintro parte in alta eroarea in loguri. Trebuie sa decidem la ce nivel o logam, sau daca o logam peste tot sa anuntam pe toti, eu de exemplu la security am inteles ca numai in controlere o logam. User doesn`t exist*